cmd/kyma/apply/function/opts.go

```diff
@@ -0,0 +1,48 @@
+package function
+
+import (
+	"github.com/kyma-project/cli/internal/cli"
+)
+
+//Options defines available options for the command
+type Options struct {
+	*cli.Options
+
+	OnError  OnError
+	Output   Output
+	Filename string
+	DryRun   bool
+}
+
+type OnError = string
+
+const (
+	NothingOnError OnError = "nothing"
+	PurgeOnError   OnError = "purge"
+)
+
+var validOnError = []string{
+	NothingOnError,
+	PurgeOnError,
+}
+
+type Output = string
```

Comment:
```suggestion
type Output string
```

Reply:
Why? This is only an alias to the string, to clarify the code

Reply:
Type aliases should not be used in regular code. They were introduced in Golang to do codebase refactoring as temporary in between for old/new types migration.

See: https://yourbasic.org/golang/type-alias/ 
And here detailed explanation: https://talks.golang.org/2016/refactor.article